repos.yaml

```diff
@@ -1543,5 +1543,10 @@ repositories:
   - name: vulcanlink
     url: https://vulcanlink.github.io/charts/
     maintainers:
-      - name: Leo Vigna 
-        email: leo@vulcan.link 
+      - name: Leo Vigna
+        email: leo@vulcan.link
+  - name: descheduler
+    url: https://kubernetes-sigs.github.io/descheduler/
+    maintainers:
+      - name: Kubernetes SIG Scheduling
+        email: kubernetes-sig-scheduling@googlegroups.com
```

Comment:
@stevehipwell I would love to include descheduler in the Helm Hub. But, I don't see you listed as a chair of sig-scheduling nor a maintainer on descheduler. I would like to have one of the owners verify they are ok being listed as a maintainer prior to merging.

Reply:
I'm a descheduler maintainer, gave my +1 above :)

@seanmalloy and @ingvagabund can also add their approvals if you'd like

Reply:
@damemi Somehow I missed that. Sorry about that. My bad.

Reply:
I'm good with this.